### ISSUE TITLE
Create CSV from HTML forms using PHP

### ISSUE THREAD

**Issue by colbySullivan** [MuellMark/Course-Scheduler#21]
I know Mark started closing and organizing issues but I promise this one should be closed very quickly. Essentially I'm going to try and create a scalable page using PHP that can save user inputs from a form.

**Comment by colbySullivan** [comment]
I started working and researching on the possibility of using PHP and pyscript concurrently. It seems very viable that we could do just that for our actually project as you can call python scripts in a PHP file using pyscript. And from the database class we learned how easy it was to connect our mysql database to said PHP file so that will make that section easier to implement.

**Comment by colbySullivan** [comment]
I got the dynamic dropdown form working with PHP where it can take values from a CSV file. I also got the input checking working as well which lets the user leave it blank and it won't break the page. The actually HTML is largely unchanged so the CSS work that Yayha is doing shouldn't have any conflicts. I'm going to work on editing the CSV form based on user inputs. Since this is PHP it should also be noted that it shouldn't take much work to convert the CSV to connect a MySQL database.

**Comment by colbySullivan** [comment]
Got a working prototype for importing and displaying a CSV file using PHP. I haven't added any of the quality checks to make sure the user imports the right file type but it works as expected.
![image](https://github.com/MuellMark/Course-Scheduler/assets/88957296/0c703490-d916-4ac4-8103-254b6d0bc537)

**Comment by colbySullivan** [comment]
I got a decently cleaned up source folder that holds a collection of all of our prototypes and connects them via hyperlinks. I am still working on a prototype page that lets you modify an existing csv. I'll merge them tomorrow when I add some more documentation.

**Comment by colbySullivan** [comment]
I am going to start work on adding a page that will edit a CSV that is on your file system then I will close this issue. Although we will not use this method directly it might prove beneficial when we get the database connected.

**Comment by colbySullivan** [comment]
I ran into an unexpected road block. The method I got working for interacting with a CSV requires it to be imported on the virtual level which means it can only be read not edited. So it will require us to use a database if we want to allow our users to edit and save their own CSV. I'm going to leave my work in the branch in case if anyone want to add to it.
https://www.cloudways.com/blog/import-export-csv-using-php-and-mysql/
